### PR TITLE
Implemented Transfer Learning in CNN_vs_ANN.ipynb

### PR DESCRIPTION
**Pull request by sebaspv**
### Description

I added instructions on how to perform transfer learning with the CIFAR-10 dataset in CNN_vs_ANN.ipynb. I also added an example that gives the best test accuracy by performing transfer learning with ResNet-50.
### Checklist

- [X] I am making a proper pull request, not spam.
- [X] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

#3 
This model has the best test accuracy with 81%. However, there's space for further enhancements by using other pretrained models or by regulating ResNet's training, because it overfits quickly on the training dataset.
## Add relevant screenshot or video (if any)

(Add here)

#### P.S. I'm not sure why the commit shows 22 files changed. I've only modified the notebook, python, and README files in the Convolutional Neural Network folder.

### CODE REVIEW

**Comment by BlueBlaze6335** [comment]
Great job @sebaspv !! I wanted someone to use transfer learning !!
